wgengine/magicsock/debugknobs.go

```diff
@@ -47,6 +47,8 @@ var (
 	// debugRingBufferMaxSizeBytes overrides the default size of the endpoint
 	// history ringbuffer.
 	debugRingBufferMaxSizeBytes = envknob.RegisterInt("TS_DEBUG_MAGICSOCK_RING_BUFFER_MAX_SIZE_BYTES")
+	// debugPMTUD enables path MTU discovery. Currently only sets the Don't Fragment sockopt.
+	debugPMTUD = envknob.RegisterBool("TS_DEBUG_PMTUD")
```

Comment:
Maybe TS_DEBUG_PMTUD -> TS_DEBUG_ENABLE_PMTUD ? Matches some other debugknobs. But if you do you will want to update the commit message.